src/open_inwoner/scss/components/Dropdown/Dropdown.scss

```diff
@@ -11,7 +11,7 @@
     flex-direction: column;
     padding: var(--spacing-medium) 0;
     position: absolute;
-    left: 0;
+    right: 0;
```

Comment:
I don't see a reason to revert this and remove the code below imo. Testing the mobile and desktop versions in the test envs I have the expected left and right results.

![acties](https://user-images.githubusercontent.com/63058071/219416605-daf1b29e-4b00-4931-be22-758510b8f75e.png)

![files](https://user-images.githubusercontent.com/63058071/219416668-e5a83151-a720-491c-8796-2b29dcedc34f.png)

Reply:
If this is what we want for the different size screens, the code looks good to me. @alextreme can you please give us your opinion?